openml/runs/functions.py

```diff
@@ -571,7 +594,8 @@ def _create_run_from_xml(xml):
 
     files = dict()
     evaluations = dict()
-    detailed_evaluations = defaultdict(lambda: defaultdict(dict))
+    fold_evaluations = defaultdict(lambda: defaultdict(dict))
+    sample_evaluations = defaultdict(lambda: defaultdict(lambda: defaultdict(dict)))
```

Comment:
I still don't understand the difference. If there are no subsamples, the only subsample index would be `0`, thus the result could be saved only in the `sample_evaluations` field, right?